src/rotate.js

```diff
@@ -0,0 +1,77 @@
+/* global WebAnimationsPolyfillExtension internalScope */
+
+(function () {
+  function isNumeric (number) {
+    return !isNaN(number);
+  }
+
+  function rotateParse (input) {
+    var rotationUnit = '';
+
+    /* According to spec:
+       https://drafts.csswg.org/css-transforms-2/#propdef-rotate
+       unspecified rotate axis default to 0 0 1
+    */
+    if (input === undefined) {
+      return null;
+    } else if (input === 'none') {
+      rotationUnit = 'deg'; // using deg as default units when angle is 0
+      internalScope.rotationUnit = rotationUnit;
```

Comment:
This is definitely not the right way to pass data between parse() and serialize(). Just include the unit in the output of parse.